packages/wallet/src/redux/protocols/resigning/states.ts

```diff
@@ -0,0 +1,93 @@
+import { Properties as P } from '../../utils';
+
+export type ResigningState = NonTerminalState | TerminalState;
+export type ResigningStateType = ResigningState['type'];
+
+export type NonTerminalState = 
+    | ConfirmResignation
+    | WaitForOpponentConclude
+    | AcknowledgeChannelClosed
+    | Defund
+
+export type TerminalState = Success | Failure;
+
+export type FailureReason =
+  | 'NotYourTurn';
+
+export interface ConfirmResignation {
+    type: 'ConfirmResignation';
+    processId: string;
+}
+
+export interface WaitForOpponentConclude {
+    type: 'WaitForOpponentConclude';
+    processId: string;
+}
+
+export interface AcknowledgeChannelClosed {
+    type: 'AcknowledgeChannelClosed';
+    processId: string;
+}
```

Comment:
Looks like you're missing the `AcknowledgeFailure` state here (or whatever you decide to call it).

Reply:
👍 thanks